test/test_examples_3d_simplices.jl

```diff
@@ -8,15 +8,24 @@ include("test_trixi.jl")
 # pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
```

Comment:
Would lumping 2d and 3d simplicial tests together help? We separated them in a previous commit, but I'd prefer them together (since they use the same code).

Reply:
Yes, that or just put them next to some of the existing 2D/3D tests.